.dockerignore

```diff
@@ -0,0 +1,13 @@
+build/
+configure.ac
+COPYING*
+docs/
+examples/
+INSTALL*
+Jenkinsfile
+Makefile.am
+m4/
+README.md
+test/
```

Comment:
we should keep this, at least for the build step so we can run `ctest -V` to make sure the tests pass before shipping.

Reply:
i can agree with that!